keel-core/src/main/java/com/netflix/rocket/semver/shaded/DebianVersionComparator.java

```diff
@@ -38,44 +38,52 @@ private int compareVersionPart(String s0, String s1) {
     String[] mainPrerelease0 = s0.split(PRERELEASE_SPLIT);
     String[] mainPrerelease1 = s1.split(PRERELEASE_SPLIT);
     int mainComparison = compareMainPart(mainPrerelease0[0], mainPrerelease1[0]);
-    if (mainComparison == 0) {
-      if (mainPrerelease0.length == mainPrerelease1.length && mainPrerelease0.length == 2) {
-        String[] prerelease0 = mainPrerelease0[1].split(PRERELEASE_SEPARATOR);
-        String[] prerelease1 = mainPrerelease1[1].split(PRERELEASE_SEPARATOR);
-        for (int i = 0; i < prerelease0.length; i++) {
-          if (i >= prerelease1.length) {
-            if (prerelease0[i].equals("dev")) {
-              return -1;
-            } else {
-              return 1;
-            }
+    if (mainComparison != 0) {
```

Comment:
The changes in this file seem unrelated to your PR?

Reply:
This is the rocket fix for the snapshot versioning compression. It's a copy-paste from their file.

Reply:
Right. So unrelated to diffing... 😄

Reply:
yes.. just my laziness for splitting PRs :)